src/annotator/adder.tsx

```diff
@@ -280,7 +280,12 @@ export class Adder implements Destroyable {
       ...document.elementsFromPoint(left + adderWidth, top + adderHeight),
     ]);
 
-    const elementZIndex = (el: Element) => +getComputedStyle(el).zIndex;
+    const elementZIndex = (el: Element) => {
+      // The `zIndex` property value is a string whose value can either be an
+      // integer or "auto".
```

Comment:
See [spec](https://drafts.csswg.org/css2/#z-index) for possible computed values.